### PR TITLE
list contact requests, fixes #248

### LINKED ISSUE #248: List contact requests
Android has an option in the menu to see the list of contact requests. We should have parity with current features available in the android app.

### PR DESCRIPTION
**Pull request by karissa**
screenshot from 2018-12-13 15-40-18

need to finish translations

### CODE REVIEW

File: src/main/deltachat.js [hunk 1]
```diff
@@ -411,29 +411,37 @@ class DeltaChatController extends events.EventEmitter {
...
-          continue
-        }
-
-        const contact = this._dc.getContact(fromId)
-
-        chat.deaddrop = {
-          messageId,
-          contact: contact.toJson()
-        }
+        chat.deaddrop = this._deadDropMessage(messageId)
```
Comment (ralphtheninja) [inline src/main/deltachat.js]: Nice

File: src/renderer/components/dialogs/DeadDrop.js [hunk 2]
```diff
@@ -26,16 +26,18 @@ class DeadDropDialog extends React.Component {
...
   }
 
   render () {
-    const { deadDropChat } = this.props
-    var nameAndAddr = deadDropChat && deadDropChat.deaddrop.contact && deadDropChat.deaddrop.contact.nameAndAddr
+    const { deadDrop } = this.props
+    var nameAndAddr = deadDrop && deadDrop.contact && deadDrop.contact.nameAndAddr
+
     const title = 'Chat request'
     const body = `Chat with ${nameAndAddr}?`
```
Comment (ralphtheninja) [inline src/renderer/components/dialogs/DeadDrop.js]: These need translation I guess? Make a TODO for it for now if it's too much trouble.

File: src/renderer/components/Menu.js [hunk 3]
```diff
@@ -135,6 +139,11 @@ class DeltaMenu extends React.Component {
         text={tx('settingsTitle')}
         onClick={openSettings}
       />
+      <MenuItem
+        icon='person'
+        text='Contact Requests'
```
Comment (ralphtheninja) [inline src/renderer/components/Menu.js]: Needs translation.

Comment (ralphtheninja) [reply src/renderer/components/Menu.js]: Make a TODO for it.

File: src/renderer/components/dialogs/DeadDrop.js (hunk 2 above)
Comment (karissa) [reply src/renderer/components/dialogs/DeadDrop.js]: done

File: src/renderer/components/Menu.js (hunk 3 above)
Comment (karissa) [reply src/renderer/components/Menu.js]: done

**Pull request merged**